lib/dradis/plugins/netsparker/importer.rb

```diff
@@ -19,20 +19,35 @@ def import(params={})
       end
 
       @doc.xpath('/netsparker/vulnerability').each do |xml_vuln|
-        process_vulnerability(xml_vuln)
+        # Create Nodes from the <url> tags
```

Comment:
@etdsoft, unfortunately, it doesn't look like Netsparker includes the IP address in the XML output. I checked all the sample files I have and none of them include an IP ☹️